.pre-commit-config.yaml

```diff
@@ -59,3 +59,18 @@ repos:
           - -w      # autofix in place
           - -d      # show diff
           - -i=2    # indent 2 spaces
+
+  # Example usage of dotnet-format: pre-commit run dotnet-format --all-files
+  - repo: local
+    hooks:
+      - id: dotnet-format
+        name: dotnet-format
+        entry: script/check-dotnet-format.sh
+        language: script
+        types:
```

Comment:
Does the python identify library have a `csharp` type? That might make the files filter unnecessary.

https://pre-commit.com/#filtering-files-with-types

Reply:
Sadly it does not

Reply:
https://github.com/chriskuehl/identify/blob/master/identify/extensions.py#L29

Reply:
Just changed - apologies that I didn't find that when googling